### DISCUSSION TITLE
Docker Image Access Denied

### DISCUSSION THREAD

**Discussion by kavindu-athaudha** [open-webui/open-webui#6159] [Q&A] [answered]
# Help Requested

## Installation Method

I used the following command to install the project through Docker:
```
docker run -d -p 3000:8080 -e OPENAI_API_KEY=your_secret_key -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```

## Environment

- **Open WebUI Version:** [main]
- **Ollama (if applicable):** [N/A]

- **Operating System:** Ubuntu 24.04.1 LTS
- **Browser (if applicable):** [N/A]

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

I expected the Docker image `ghcr.io/open-webui/open-webui:main` to be pulled successfully and the container to start running.

## Actual Behavior:

Upon attempting to run the command, I encountered the am error message suggesting that the image is not accessible, possibly indicating it is set to private.

## Description

**Bug Summary:**
The Docker image `ghcr.io/open-webui/open-webui:main` appears to be private, leading to a permission denied error when trying to pull it.

## Reproduction Details

**Steps to Reproduce:**
1. Open a terminal in your operating system.
2. Run the following command:
   ```
   docker run -d -p 3000:8080 -e OPENAI_API_KEY=your_secret_key -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
   ```
3. Observe the error returned by Docker regarding the inability to find the image.

## Logs and Screenshots

**Docker Container Logs:**
```
Unable to find image 'ghcr.io/open-webui/open-webui:main' locally
docker: Error response from daemon: Head "https://ghcr.io/v2/open-webui/open-webui/manifests/main": denied: denied.
See 'docker run --help'.
```

**Comment by defel** [comment, marked as answer]
try this: `docker logout ghcr.io`

**Reply by kavindu-athaudha** [reply]
That worked!
Thank you very much.

**Reply by guilbep** [reply]
Wow. Thanks, that was crazy.  Looks like fine grained and authorization tokens are messing with this.
I found out somewhere in the doc that logging in can override the default public access settings, causing conflicts because of fine grained tokens or something

I guess anonymous vs logged in make it weird

**Reply by ebenoist** [reply]
Yup, hit the same issue here. Thanks for the solution!

**Reply by olang221** [reply]
Super, vielen Dank!!

**Reply by Promathazinee** [reply]
I am still running into the sam error

**Reply by Promathazinee** [reply]
Removing login credentials for ghcr.io
Unable to find image 'ghcr.io/open-webui/open-webui:main' locally

**Reply by noodlesTT** [reply]
It doesn't worked.I am still running into the same error.

**Reply by Toasterme** [reply]
same, does not work still : 
Failed recreating container: Pull image error ghcr.io/open-webui/open-webui:latest: context deadline exceeded (Client.Timeout or context cancellation while reading body)

Using portainer.

**Reply by chiabdel** [reply]
I have the same issue :
Unable to find image 'ghcr.io/open-webui/open-webui:main' locally
docker: Error response from daemon: Get "https://ghcr.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers).
See 'docker run --help'.

Any help ?

**Comment by leviethung2103** [comment]
Thank you so much. It worked

**Comment by gmahe** [comment]
I've restarted docker and did `docker pull ghcr.io/open-webui/open-webui:main` and it worked
